meta.md

```diff
@@ -44,8 +44,8 @@ This PER shares the same goals as PSR-12.
 > of guidelines to be used among all those projects. Thus, the benefit of this guide is
 > not in the rules themselves, but in the sharing of those rules.
 
-This PER is an extension of PSR-12, and therefore also an extension of PSR-1. The basis of PER is
-PSR-12 and therefore a list of differences is provided below to assist with migration,
+This PER is an extension of PSR-12, and therefore also an extension of PSR-1. 
+With PSR-12 being the base of this PER, a list of differences is provided below to assist with migration, 
```

Comment:
Can you remove this trailing space? Other than that it looks good to me

Reply:
done.